dynawo/sources/Models/Modelica/Dynawo/Examples/IEEE118/BaseClasses/IEEE118Base.mo

```diff
@@ -0,0 +1,1111 @@
+    Dynawo.Electrical.Machines.SignalN.GeneratorPVTfo B112_G1(KGover = 0.0, PGen0Pu = 0.0, PMaxPu = 100.0, PMinPu = -100.0, PNom = 100.0, PRef0Pu = -0.0, QGen0Pu = 0.4151, QMaxPu = 10.0, QMinPu = -1.0, QNomAlt = 1000.0, QStator0Pu = 0.0415, SNom = 10050.0, U0Pu = 1.0, UStator0Pu = 1.0005, UStatorRef0Pu = 0.9755, XTfoPu = 0.1228, SwitchOffSignal10 = false, SwitchOffSignal20 = false, SwitchOffSignal30 = false, limUQDown0 = false, limUQUp0 = false, qStatus0 = Dynawo.Electrical.Machines.SignalN.GeneratorPV.QStatus.Standard, i0Pu = Complex(0.0, 0.4151), iStator0Pu = Complex(0.0, -0.0041), sStator0Pu = Complex(0.0, 0.0041), u0Pu = Complex(1.0, 0.0), uStator0Pu = Complex(1.0005, 0.0));
+    Dynawo.Electrical.Machines.SignalN.GeneratorPVTfo B113_G1(KGover = 0.0, PGen0Pu = 0.0, PMaxPu = 100.0, PMinPu = -100.0, PNom = 100.0, PRef0Pu = -0.0, QGen0Pu = 0.0632, QMaxPu = 2.0, QMinPu = -1.0, QNomAlt = 200.0, QStator0Pu = 0.0316, SNom = 10002.0, U0Pu = 1.0, UStator0Pu = 1.0001, UStatorRef0Pu = 0.9931, XTfoPu = 0.1228, SwitchOffSignal10 = false, SwitchOffSignal20 = false, SwitchOffSignal30 = false, limUQDown0 = false, limUQUp0 = false, qStatus0 = Dynawo.Electrical.Machines.SignalN.GeneratorPV.QStatus.Standard, i0Pu = Complex(0.0, 0.0632), iStator0Pu = Complex(0.0, -0.0006), sStator0Pu = Complex(0.0, 0.0006), u0Pu = Complex(1.0, 0.0), uStator0Pu = Complex(1.0001, 0.0));
+    Dynawo.Electrical.Machines.SignalN.GeneratorPVTfo B116_G1(KGover = 0.0, PGen0Pu = 0.0, PMaxPu = 100.0, PMinPu = -100.0, PNom = 100.0, PRef0Pu = -0.0, QGen0Pu = 0.5132, QMaxPu = 10.0, QMinPu = -10.0, QNomAlt = 1000.0, QStator0Pu = 0.0514, SNom = 10050.0, U0Pu = 1.0, UStator0Pu = 1.0006, UStatorRef0Pu = 1.0056, XTfoPu = 0.1228, SwitchOffSignal10 = false, SwitchOffSignal20 = false, SwitchOffSignal30 = false, limUQDown0 = false, limUQUp0 = false, qStatus0 = Dynawo.Electrical.Machines.SignalN.GeneratorPV.QStatus.Standard, i0Pu = Complex(0.0, 0.5132), iStator0Pu = Complex(0.0, -0.0051), sStator0Pu = Complex(0.0, 0.0051), u0Pu = Complex(1.0, 0.0), uStator0Pu = Complex(1.0006, 0.0));
+
+     //Loads
+    Dynawo.Electrical.Loads.LoadAlphaBetaRestorative B1_L1(Alpha = 1.5, Beta = 2.5, UMaxPu = 1.15, UMinPu = 0.85, i0Pu = Complex(0.4536, -0.3258), s0Pu = Complex(0.476, 0.2407), tFilter = 10.0, u0Pu = Complex(0.9437, -0.1471));
```

Comment:
Since all values of Alpha, Beta, UMinPu, UMaxPu and filter are the same, you could define parameters like what's done in IEEE14 : 
parameter Real Alpha = 1.5 "Active load sensitivity to voltage";
parameter Real Beta = 2.5 "Reactive load sensitivity to voltage";
parameter Types.VoltageModulePu UMaxPu = 1.05 "Maximum value of the voltage amplitude at terminal in pu (base UNom) that ensures the P/Q restoration";
parameter Types.VoltageModulePu UMinPu = 0.95 "Minimum value of the voltage amplitude at terminal in pu (base UNom) that ensures the P/Q restoration";
parameter Types.Time tFilter = 10;